### ISSUE TITLE
clippy issues in the generated shadow.rs file

### ISSUE THREAD

**Issue by szabgab** [baoyachi/shadow-rs#151]
I have a project with lots of clippy lints enables. Adding `shadow-rs` I started to get lots of complaints from the `out/shadow.rs` that, as I understand it, is generated during the build step. Specifically I had to disable these.

```
[lints.clippy]
non_ascii_literal = "allow"
print_stdout = "allow"
needless_raw_strings = "allow"
needless_raw_string_hashes = "allow"
```

It would be nice if I could enable them again.

Using shadow-rs  version 0.26.1

**Comment by dsully** [comment]
+1

**Comment by thecaralice** [comment]
This issue was fixed for me when I switched to `0.32.0`, try that maybe

**Comment by baoyachi** [comment]
Thank you for your feedback on this issue. Although the issue has been pending for a while, we still appreciate @thecaralice for resolving it.

Please use new publised version:
```toml
shadow-rs = "0.33.0"
```
